prow/cmd/peribolos/main.go

```diff
@@ -61,7 +61,12 @@ type options struct {
 	ignoreSecretTeams bool
 	allowRepoArchival bool
 	allowRepoPublish  bool
-	github            flagutil.GitHubOptions
+	// TODO(MadhavJivrajani): this is a temporary mitigation to help
+	// fix a configuration mishap:
+	// https://kubernetes.slack.com/archives/CHGFYJVAN/p1690906660582369
+	// There is no real use for this, remove once things are in good shape.
+	skipRemovals bool
```

Comment:
TODO's can be easy to leave, can we print a warning when the flag is used that this won't be supported long term?

And/or hide the flag from usage / mark deprecated (I don't think we're using libraries that support that, so the warning is sufficient if not)